README.md

```diff
@@ -1,6 +1,6 @@
 # Google Student Training in Engineering Program
 
-This repo contains the projects you'll work on for the first half of STEP.
+This file contains Amy Nguyen's portfolio and STEP projects
 This work is based on the [Google Software Product Sprint](https://g.co/softwareproductsprint) program.
 
 To get started:
```

Comment:
I think this can be removed.

Reply:
You marked this conversation as resolved. Is there a reason why?

Reply:
Sorry, I thought you were referring to a different line that was already removed. I'll go ahead and remove the lines for "To get started"

Reply:
Not sure if you noticed, but I highlighted the lines I was referring to in the comment. This add context as to what I am doing. Another tip is to comment on the open comments before resolving them. Even if it is just to say you: Done.